py3dtilers/GeojsonTiler/geojson.py

```diff
@@ -154,29 +125,29 @@ def parse_geom(self):
         # ]
 
         coords = self.coords
-        height = self.height
+        height = self.height # How high we extrude the polygon when creating the 3D geometry
+        z = self.z # Altitude of the polygon that will be extruded to create the 3D geometry
 
         # If the feature has at least 4 coords, create a convex hull
         # The convex hull reduces the number of points and the level of detail
         if len(coords) >= 4:
-            # coords = rdp(coords)
-            hull = ConvexHull(coords)
-            coords = [coords[i] for i in reversed(hull.vertices)]
+            hull = alphashape(coords, 0.)
+            coords = hull.exterior.coords[:-1]
+            # hull = ConvexHull(coords)
+            # coords = [coords[i] for i in reversed(hull.vertices)]
 
         coordsLenght = len(coords)
-        vertices = np.ndarray(shape=(2 * (coordsLenght + 1), 3))
+        vertices = [None] * (2 * (coordsLenght + 1))
```

Comment:
I need a brief conversation on the reason for loosening types away from `np` types (e.g this `np.ndarray`)...

Reply:
I tried to align with [py3dtiles triangulation](https://github.com/VCityTeam/py3dtiles/blob/master/py3dtiles/wkb_utils.py#L12)  types, where:
- the geometry is a list of triangles
- each triangle is a list of vertices
- each vertice is a np.array containing the [x, y, z] coordinates